integreat_cms/cms/templates/chat/_chat_messages.html

```diff
@@ -2,6 +2,7 @@
 {% load static %}
 {% load rules %}
 {% for message in chat_messages %}
+<<<<<<< HEAD
```

Comment:
Please resolve the git conflicts manually by either taking the `HEAD` version and making your change with replacing the newline tag again or take your `newline in chat deletion confirmation box` state and format the template with djlint.

Reply:
Ah yeah sorry, you already fixed it :grin:

Reply:
Yes but I think I need to fix again this generic_confirmation_dialog.html. As I see that generated translation in django is connected to the lines. I try one more time to change it.

Reply:
You can just run `./dev-tools/djlint.sh` to apply our new HTML formatting style.